### ISSUE TITLE
OUTPUT_DIR with the gcs path does not work

### ISSUE THREAD

**Issue by yshrkume** [blue-oil/blueoil#1121]
`os.path.abspath()` removes one of the path slashes.

```
$ python
Python 3.6.9 (default, Jul  3 2019, 15:36:16)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> path = 'gs://hoge/fuga/piyo/'
>>> OUTPUT_DIR = os.path.abspath(path)
>>> OUTPUT_DIR
'/home/User/workingdir/gs:/hoge/fuga/piyo'
```

https://github.com/blue-oil/blueoil/blob/r0.25/blueoil/environment.py#L42

**Comment by iizukak** [comment]
@yshrkume 
Thank you for the report.
We will fix soon.

**Comment by iizukak** [comment]
Fixed with #1122
